packages/grid_client/scripts/utils.ts

```diff
@@ -3,4 +3,17 @@ import { inspect } from "util";
 function log(message) {
   console.log(inspect(message, { showHidden: false, depth: null, colors: true }));
 }
-export { log };
+
+async function pingNodes(client, nodes) {
+  for (const node of nodes) {
+    try {
+      await client.zos.pingNode({ nodeId: node.nodeId });
+      return node.nodeId;
```

Comment:
if you prefer to return the `nodeID` instead of the node interface, please define it in the function return type.